sshsysmon/inspectors/gpu.py

```diff
@@ -0,0 +1,52 @@
+import logging
```

Comment:
Looks like a few things are imported aren't used. Can you clean up if able?

Reply:
Sure